.github/workflows/standalone.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
     - main
+    - 'debug-standalone-build'
```

Comment:
do we want to keep this or remove it?  Or maybe some other wildcard prefix so we don't have to edit main in the future?

Reply:
Is this branch to stay? I thought it was a temporary branch.

Reply:
Also, do we want the option to enable in PR using special label, or not worth it?